wled00/e131.cpp

```diff
@@ -346,7 +346,6 @@ void handleArtnetPollReply(IPAddress ipAddress) {
 
   switch (DMXMode) {
     case DMX_MODE_DISABLED:
-      return;  // nothing to do
```

Comment:
while this code may well be redundant, code cleanup in a PR that is meant to be adding new features muddies the waters a bit. Better to keep the PR to only the changes required for the task

Reply:
I replaced the return with an if condition because otherwise the code I added wouldn't be called if DMX mode is set to disabled

Reply:
ah, yes I follow what you are doing now